packages/bits/src/lib/time-picker/time-picker.component.ts

```diff
@@ -142,14 +152,34 @@ export class TimePickerComponent implements OnInit, OnDestroy, OnChanges, AfterV
         this.overlay.clickOutside
             .pipe(takeUntil(this.onDestroy$))
             .subscribe(_ => this.overlay.hide());
-        
+
         this.initPopupUtilities();
+        this.keyboardService.initService(this.popup.menuItems, this.popup, this.overlay, this.menuTrigger.nativeElement);
```

Comment:
if we're passing in the `this.popup`, we can probably retrieve the `this.popup.menuItems` from it later, can't we?